model/openapi.yml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.0
+openapi: 3.1.0
```

Comment:
Note that the spec version has been upgraded and some minor changes [following the migration guide](https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0) have been made.